app/products/calls/state/actions.ts

```diff
@@ -233,6 +237,19 @@ export const callStarted = (serverUrl: string, call: Call) => {
         ...call,
     };
     setCurrentCall(nextCurrentCall);
+
+    // We started the call, and it succeeded, so follow the call thread.
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return;
+    }
+
+    // Make sure the post/thread has arrived from the server.
+    const thread = await getThreadById(database, call.threadId);
+    if (thread && !thread.isFollowing) {
```

Comment:
same here, check if CRT is enabled. in fact you could check for CRT before querying the thread.